packages/yoga/src/Divider/web/Divider.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import theme from '../../Theme/helpers/themeReader';
+
+const StyledDivider = styled.div`
+  ${props => {
+    const { borders, colors } = theme(props);
+
+    return css`
+      flex: 1;
+      height: ${borders.small}px;
+      background-color: ${colors.light};
+    `;
+  }}
+`;
+
+const Divider = props => {
+  return <StyledDivider {...props} />;
+};
+
+Divider.propTypes = {};
+
+Divider.defaultProps = {};
```

Comment:
Same comments about the need for some props and implementing some `system` props.